charts/move2kube/values.schema.json

```diff
@@ -0,0 +1,184 @@
+            "examples": [{
+                "name": "serverless-workflow-m2k",
+                "image": "quay.io/orchestrator/serverless-workflow-move2kube:latest",
+                "brokerURL": "http://broker-ingress.knative-eventing.svc.cluster.local/sonataflow-infra/default",
+                "move2kubeURL": "https://move2kube-route-sonataflow-infra.apps.cluster-8xfw.redhatworkshops.io",
+                "backstageNotificationURL": "http://orchestrator-backstage.orchestrator/api/notifications/"
+            }]
+        },
+        "kfunction": {
+            "type": "object",
+            "default": {},
+            "title": "The kfunction Schema",
+            "required": [
+                "name",
+                "image"
+            ],
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The name Schema",
+                    "examples": [
+                        "m2k-save-transformation-func"
+                    ]
+                },
+                "image": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The image Schema",
+                    "examples": [
+                        "quay.io/orchestrator/serverless-workflow-m2k-kfunc:latest"
+                    ]
+                }
+            },
+            "examples": [{
+                "name": "m2k-save-transformation-func",
+                "image": "quay.io/orchestrator/serverless-workflow-m2k-kfunc:latest"
+            }]
+        },
+        "instance": {
+            "type": "object",
+            "default": {},
+            "title": "The instance Schema",
+            "required": [
+                "name",
+                "image"
+            ],
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The name Schema",
+                    "examples": [
+                        "move2kube"
+                    ]
+                },
+                "image": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The image Schema",
+                    "examples": [
+                        "quay.io/orchestrator/move2kube-ui:latest"
+                    ]
+                }
+            },
+            "examples": [{
+                "name": "move2kube",
+                "image": "quay.io/orchestrator/move2kube-ui:latest"
+            }]
+        }
+    },
+    "examples": [{
+        "sshSecretName": "sshkeys",
+        "brokerName": "default",
+        "namespace": "sonataflow-infra",
+        "workflow": {
+            "name": "serverless-workflow-m2k",
+            "image": "quay.io/orchestrator/serverless-workflow-move2kube:latest",
+            "brokerURL": "http://broker-ingress.knative-eventing.svc.cluster.local/sonataflow-infra/default",
```

Comment:
no need for broker URL, it's hard coded in the workflow, we should remove it from the values